doc/source/changed_in_version.rst

```diff
@@ -149,3 +149,7 @@ Removed System variables
   which was an alias to the ``kill_idle_trx`` system variable
 - The `max_binlog_files <https://www.percona.com/doc/percona-server/5.7/flexibility/max_binlog_files.html#max_binlog_files>`_ system variable
 
+Deprecated Storage engine
+-----------------------------------------------------
+
+- TokuDB is deprecated in the 8.0 series and, after version 8.0.25, is not supported. Percona will remove this storage engine in the next major release of Percona Server for MySQL. We recommend MyRocks as a long-term migration path.
```

Comment:
The exact wording and time line is still under debate. Please withhold from merging this PR for now.

Reply:
Updated the notes on other pages to match the RN. Added migration steps to Removing TokuDB.